client/src/App.js

```diff
@@ -26,85 +28,80 @@ class App extends Component {
       <div>
         <Navbar fluid>
           <Navbar.Header>
```

Comment:
I can do some more googling and monkeying with CSS to figure it out, but what is the Navbar styling doing that you already had in place? Because I can't get anything to align to the right side of the page and I can't figure out what is keeping it from working, lol. I'll keep messing with it, but just figured I'd ask anyway.